TABLE 3. Mapping of transposon insertions sites that result in white phenotype in *C. sakazakii* ES5

| COG functional category <sup>a</sup>                                      | COG functional class                                                                                                  | Annotation <sup>b,c</sup>            |                                                                                                      |
|---------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------|--------------------------------------|------------------------------------------------------------------------------------------------------|
|                                                                           |                                                                                                                       | Homologue                            | Gene product                                                                                         |
| Mutation in pigment operon                                                |                                                                                                                       |                                      |                                                                                                      |
| H: coenzyme transport and metabolism                                      | Geranylgeranyl pyrophosphate synthase                                                                                 | crtE                                 | Geranylgeranyl pyrophosphate synthase                                                                |
| GC: carbohydrate transport and metabolism/signal transduction mechanisms  | Glucosyl transferases, related to UDP-glucosyltransferase                                                             | crtX                                 | Zeaxanthin glucosyl transferase                                                                      |
| R/E: general function prediction only/amino acid transport and metabolism | Acetyltransferase/choline dehydrogenase and related flavoproteins                                                     | crtY                                 | Lycopene cyclase                                                                                     |
| Q: secondary metabolites biosynthesis, transport and catabolism           | Phytoene dehydrogenase and related proteins                                                                           | crtI                                 | Phytoene dehydrogenase                                                                               |
| I: lipid transport and metabolism                                         | Phytoene/squalene synthase                                                                                            | crtB                                 | Phytoene synthase                                                                                    |
| Mutation outside pigment operon                                           |                                                                                                                       |                                      |                                                                                                      |
| C: energy production and conversion                                       | F <sub>o</sub> F <sub>1</sub> -type ATP synthase, subunit alpha                                                       | ESA_04012                            | F <sub>o</sub> F <sub>1</sub> ATP synthase subunit alpha                                             |
|                                                                           | F <sub>o</sub> F <sub>1</sub> -type ATP synthase, subunit beta                                                        | ESA_04006                            | F <sub>o</sub> F <sub>1</sub> ATP synthase subunit beta                                              |
|                                                                           | F <sub>o</sub> F <sub>1</sub> -type ATP synthase, subunit gamma                                                       | ESA_04007                            | F <sub>o</sub> F <sub>1</sub> ATP synthase subunit gamma                                             |
|                                                                           | F <sub>o</sub> F <sub>1</sub> -type ATP synthase, subunit epsilon (mitochondrial delta subunit)                       | ESA_04005                            | F <sub>o</sub> F <sub>1</sub> ATP synthase subunit epsilon                                           |
|                                                                           | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E1) component, and related enzymes | ESA_02622                            | sucA 2-oxoglutarate dehydrogenase E1 component                                                       |
|                                                                           | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E2) component, and related enzymes | ESA_02621                            | Dihydrolipoamide acetyltransferase                                                                   |
|                                                                           | Pyruvate/2-oxoglutarate dehydrogenase complex, dihydrolipoamide acetyltransferase (E3) component, and related enzymes | ESA_03222                            | aceF dihydrolipoamide acetyltransferase                                                              |
|                                                                           | Malate/lactate dehydrogenases                                                                                         | ESA_03622                            | Malate dehydrogenase                                                                                 |
|                                                                           | Succinate dehydrogenase/fumarate reductase, flavoprotein subunit                                                      | ESA_02624                            | Succinate dehydrogenase flavoprotein subunit                                                         |
| P: inorganic ion transport and metabolism                                 | Na <sup>+</sup> /H <sup>+</sup> antiporter                                                                            | ESA_03316                            | pH-dependent sodium/proton antiporter                                                                |
| T: signal transduction mechanisms                                         | cAMP-binding proteins, catabolite gene activator, and regulatory subunit of cAMP-dependent protein kinases            | ESA_04376                            | cAMP regulatory protein                                                                              |
|                                                                           | DnaK suppressor protein                                                                                               | ESA_03194                            | DnaK transcriptional regulator DksA                                                                  |
| S: function unknown                                                       | Uncharacterized conserved protein                                                                                     | ESA_04343 (Ent638_3811) <sup>d</sup> | Hypothetical protein (intracellular growth attenuator IgA, <i>Enterobacter</i> sp. 638) <sup>d</sup> |
|                                                                           |                                                                                                                       | ESA_03563 (ETA_03450) <sup>d</sup>   | Hypothetical protein (Yhbc-like protein, <i>Erwinia tasmaniensis</i> Et1/99) <sup>d</sup>            |
|                                                                           |                                                                                                                       | ESA_00549 (AAG53883) <sup>d</sup>    | Hypothetical protein (sigma factor RpoS, <i>Escherichia coli</i> ) <sup>d</sup>                      |

<sup>a</sup> NCBI clusters of orthologous groups (COG) of proteins.<sup>b</sup> *Cronobacter sakazakii* ES5 BAC 9E10 for mutations within the pigment operon (accession no. AM384990.1).<sup>c</sup> NCBI assembly ATCC BAA-894 *C. sakazakii* complete genome for mutations outside pigment operon (accession no. CP000783.1).<sup>d</sup> Closest annotated homolog.(data not shown), consistently with the results for LB, all mutant strains showed significantly increased maximum rates of growth ( $\mu_{\max, \text{wt}}$ , 0.14;  $\mu_{\max, \Delta c r t X}$ , 0.22;  $\mu_{\max, \Delta c r t E}$ , 0.24; and  $\mu_{\max, \Delta c r t Y}$ , 0.19;  $P = 0.000$ ).

Cold stress experiments were performed by growing  $\Delta c r t E$ ,  $\Delta c r t X$ , and  $\Delta c r t Y$  in LB and M9 medium at 10Â°C (for results in M9, see Fig. 1B). Under these conditions, no significant dif-

ferences in maximum specific growth rates were detected for  $\Delta c r t Y$ ,  $\Delta c r t E$ , and  $\Delta c r t X$  in both LB and M9 compared to those of the wild type (in LB,  $\mu_{\max, \text{wt}} = 0.04$ ,  $\mu_{\max, \Delta c r t X} = 0.03$ ,  $\mu_{\max, \Delta c r t E} = 0.03$ , and  $\mu_{\max, \Delta c r t Y} = 0.04$ ; in M9,  $\mu_{\max, \text{wt}} = 0.01$ ,  $\mu_{\max, \Delta c r t X} = 0.01$ ,  $\mu_{\max, \Delta c r t E} = 0.01$ , and  $\mu_{\max, \Delta c r t Y} = 0.01$ ).

To evaluate growth under acidic conditions, wild-type and